addons/l10n_cl/models/account_move_line.py

```diff
@@ -65,15 +65,15 @@ def _l10n_cl_get_line_amounts(self):
             second_currency_field = 'price_subtotal'
             second_currency = self.currency_id
             main_currency_rate = 1
-            second_currency_rate = abs(self.balance) / self.price_subtotal if domestic_invoice_other_currency else False
+            second_currency_rate = main_currency._get_conversion_rate(second_currency, main_currency, self.company_id, self.date)
```

Comment:
No, the idea is to not use that, because they could be altered before/after the invoice was generated.  We could do it on the totals of the invoice instead of the line totals?

Reply:
Yes. I can do something like this to keep the same rate for all lines:
`second_currency_rate = abs(self.move_id.amount_total_signed) / self.move_id.amount_total if self.move_id.amount_total else 1`

But it's weird because the displayed rate is not the one that is configured on the currency.